src/fonduer/utils/utils_udf.py

```diff
@@ -29,6 +29,8 @@
 # integer to ensure that it won't conflict with a user's split value.
 ALL_SPLITS = "ALL"
 
+ABSTAIN = -1
```

Comment:
Should this be in the labeler since it's labeler specific?

Reply:
I thought so at first, but soon realized that it'd cause a circular imports between labeler and utils_udf.

Reply:
Done at 6e7f16965e298dab1fe2702ef968cbe66b842333.